### ISSUE TITLE
Resizable textareas

### ISSUE THREAD

**Issue by bemoody** [MIT-LCP/physionet-build#384]
There's a CSS rule somewhere that sets textareas to "resize: vertical", which means "don't let the user resize the box horizontally".  Rather obnoxious; I didn't know CSS had such a "feature".

Of course it'd be good to set the default size to something sensible, but if user prefers to resize the box to something larger (or smaller) they shouldn't be stopped from doing so.

**Comment by Lucas-Mc** [comment]
It is certainly possible to remove the `resize: vertical` attribute, but I think we should prevent the box from being wider than the box it is in with some degree of padding. Further, for example, if there is a text box above it, then it shouldn't exceed its width. 

I think the purpose of this property is to maintain the aesthetic of the webpage while still providing  the option to change the box size to view more text (The current text box I am typing in on Github has this same property). Can you think of a good reason why we should break this? Or why someone would want to have a skinnier text box? @tompollard What are your thoughts?

**Comment by bemoody** [comment]
Hmm, well, I tend to find editing anything in the browser to be an
uncomfortable experience in general, and if I have to write something
more than a paragraph or two of plain text, I'll do it in emacs and
copy and paste into the browser.

But I do think it'd be good to try to make things easier on authors.
I like that newer browsers normally allow you to resize textareas,
because the size that the web designer thought was aesthetically
pleasing is usually far too small.

The best size is very much a matter of personal preference and depends
on what you're editing and how large your screen is - longer lines let
you have more text on screen at once, but also make it harder to read.

**Comment by Lucas-Mc** [comment]
Yeah, I think it's a tough question to answer since it's hard to tell how many people are writing out paragraphs in there (I sure hope not) or just copy and pasting. Maybe we can test it out without `resize: vertical` enabled to see if we like it better? Hopefully it won't cause any formatting issues with the other elements that text box interacts with; I don't think it will but it's something to look out for.
